_posts/2020-10-01-annual-report-part-1.md

```diff
@@ -0,0 +1,58 @@
+- **July 2020**: OLS-2 call for application opens
+- **September 2020**: OLS-2 launched with 32 participating projects
+- **December 2020**: OLS-2 expected graduation
+- **December 2020**: OLS-3 call for applications is expected to open
+- **February 2021**: OLS-3 cohort is expected to start
+
+## Inclusiveness designed at the core of OLS
+
+We designed our project to be inclusive of different demographics, by intentionally targeting researchers from countries, identities and genders who are traditionally underrepresented in Open Science (thanks to [166 members in OLS community]({% link community.md %})). We personally reached out to all the mentors and experts from diverse domain knowledge, personal backgrounds and identities. Our current [mentors](/ols-2#mentors) represent scientific communities from Australia, Brazil, Canada, China, Ethiopia, France, Germany, Ghana, Kenya, South Africa, Spain, Switzerland, The Netherlands, UK, and the USA. Our [expert community](/ols-2#experts) is double in both size and representation. We started our [first cohort]({% link _ols-1/projects-participants.md %}) with mentees who came from equally diverse countries from 5 continents, representing low, middle and high economy countries equally. [29 individuals](/ols-1/projects-participants/#participants) joined from Kenya, Netherlands, Brazil, Canada, Thailand, Spain, UK, Japan, Russian, India, USA, Norway, Germany, and Nepal. Our mentors represented research communities from China, Greece, the UK, the USA, South Africa, Germany, Kenya, Netherlands, and Brazil.
+
+We use welcoming and open channels for community participation. We have a cohort-based Slack channel and a public-facing Gitter channel ([more details](/ols-2#resources)). We strive to avoid jargon and other non-inclusive languages that can alienate, and make underrepresented people feel excluded. We use simple and jargon-free English in our communications, website and training resources (see an example from [one of the cohort calls]({% link _ols-1/week04.md %})). On our website, pronouns are listed for every [participant in their profile](/ols-1/projects-participants/#participants). At the beginning of each cohort call, we remind our [Code of Conduct]({% link code-of-conduct.md %}) and use roll call where participants are asked to list their names and pronouns so others address them respectfully. We encourage and recognize the quietest voices, and not just those with the most confidence, and volume. During the cohort calls, we facilitate breakout discussions and encourage silent note-taking in shared notes. In OLS-2, cohort calls are live-transcribed using [Otter.ai](https://otter.ai), and some breakout rooms are non-verbal, to ensure that people with varying language skills or ability can still get an interactive and full experience. Recordings from these calls are posted on YouTube for the cohort members to catch up if they couldn't attend the call.
```

Comment:
```suggestion
We use welcoming and open channels for community participation. We have a cohort-based Slack channel and a public-facing Gitter channel ([more details](/ols-2#resources)). We strive to avoid jargon and other non-inclusive language that can alienate, and make underrepresented people feel excluded. We use simple and jargon-free English in our communications, website and training resources (see an example from [one of the cohort calls]({% link _ols-1/week04.md %})). On our website, pronouns are listed for every [participant in their profile](/ols-1/projects-participants/#participants). At the beginning of each cohort call, we remind participants about our [Code of Conduct]({% link code-of-conduct.md %}) and use a roll call where participants are asked to list their names and pronouns so others address them respectfully. We encourage and recognize the quietest voices, and not just those with the most confidence and volume. During the cohort calls, we facilitate breakout discussions and encourage silent note-taking in shared notes. In OLS-2, cohort calls are live-transcribed using [Otter.ai](https://otter.ai), and some breakout rooms are non-verbal, to ensure that people with varying language skills or ability can still get an interactive and full experience. Recordings from these calls are posted on YouTube for the cohort members to catch up if they couldn't attend the call.
```

Reply:
❤️🌸